pyproject.toml

```diff
@@ -15,6 +15,16 @@ skip = ["ddtrace/vendor/", ".riot", ".ddriot", ".tox", ".ddtox", ".eggs", "build
 skip_glob = [".venv*", "ddtrace/profiling/exporter/pprof_*pb2.py"]
 line_length = 120
 
+[tool.cython-lint]
+max-line-length = 120
+exclude = '''
+(
+  .venv*
+  | \.riot
+  | ddtrace/profiling/
```

Comment:
why is profiling excluded?

Reply:
1. I wanted to reduce the scope for this PR
2. There are some violations I am unsure how to handle right away

Reply:
Happy to consider including profiling fixes here as well if we want!